Mupa1/Questioner
# #162985659 latest meetups

__What does this PR do?__

Adds latest meetups to landing page

__Description of task to be completed__
Have the following goals achieved:
* A responsive landing page
* Add latest meetups
* Add a search bar
* Create a link to all meetups page

__How should this be manually tested?__
1. Cloning the repo ```https://github.com/Mupa1/Questioner.git``` 
2. Go to the UI folder and open index.html on a browser

__Relevant pivotal tracker stories?__
#162985659

__Screenshots__
![landing_page](https://user-images.githubusercontent.com/44979953/50723543-67223000-10f0-11e9-8fb8-7b5d9803d579.png)
